spec/models/filter/resetFilter.spec.ts

```diff
@@ -0,0 +1,31 @@
+import {FilterModel, FilterModelProperty} from "../../../src/models/filter/filter-model";
+import {TestItem} from "./test-item";
+import {FilterConfig, FilterConfigItem} from "../../../src/models/filter/filter-config-item";
+
+describe('Filter model - resetFilter:', () => {
+    let filter: FilterModel<TestItem>;
+    let data: TestItem[] = [];
+    let testModel: FilterConfig;
+    let iqs: number [] = [];
+
+    beforeEach(() => {
+        filter = new FilterModel();
+        data = [new TestItem(), new TestItem(), new TestItem()];
+        testModel = {
+            age: new FilterConfigItem('Age'),
+            iq: new FilterConfigItem('iq')
+        };
+        data.forEach(i => iqs.push(i.iq));
+    });
+
+    it('Test 1: Base test', () => {
```

Comment:
`Should reset filter model of provided report property to the default min and max values`  
`Should do nothing if filter model is in the default state`